Nimble/Expectation.swift

```diff
@@ -8,20 +28,12 @@ public struct Expectation<T> {
     }
 
     public func to<U where U: Matcher, U.ValueType == T>(matcher: U) {
-        var msg = FailureMessage()
-        let pass = matcher.matches(expression, failureMessage: msg)
-        if msg.actualValue == "" {
-            msg.actualValue = "<\(stringify(expression.evaluate()))>"
-        }
+        let (pass, msg) = expectTo(expression, matcher, "to")
         verify(pass, msg.stringValue())
     }
 
     public func toNot<U where U: Matcher, U.ValueType == T>(matcher: U) {
-        var msg = FailureMessage()
-        let pass = matcher.doesNotMatch(expression, failureMessage: msg)
-        if msg.actualValue == "" {
-            msg.actualValue = "<\(stringify(expression.evaluate()))>"
-        }
+        let (pass, msg) = expectToNot(expression, matcher, "to not")
```

Comment:
Oh, haha, looks like the readability issue I commented on above already existed! I still stand by it, though. :sweat_smile: